Comp2023/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -199,10 +214,7 @@ private void configureButtonBindings( )
     operY.onTrue(new Dummy(XboxController.Button.kY.value));
     //
     // Operator - Bumpers, start, back
-    operLeftBumper.onTrue(new GripperRun(m_gripper, GRMode.GR_ACQUIRE));
```

Comment:
Can we put these back onto the Dummy command with onTrue? It is an obvious placeholder for the next person that the button is available, and it will print a log message if someone hits it even though it's not assigned to a useful function. (also, I thought the intaking was going to stay on the bumpers, but I may have missed that conversation today).

Reply:
Based on the images from #software it looks like the intaking is assigned to the triggers but I can change it if we want to use the bumpers.

Reply:
Because we used bumpers on Festus and Grogu, let's start there. Although they may get pushed off if we need something more important. Practice will tell.